server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1337,6 +1392,10 @@ public File exportCertificates(
             .listForConsumer(consumer)) {
             allCerts.add(new CertificateSerialDto(cert.getSerial().getId()));
         }
+        // add content access cert if needed
```

Comment:
If all this method really cares about is a list of the serials, a JPA union query could get that information in a more efficient manner.